docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  qpc-server:
+    build: .
+    environment:
+      QPC_DBMS: postgres
+      QPC_DBMS_DATABASE: postgres
+      QPC_DBMS_HOST: localhost
+      QPC_DBMS_PASSWORD: qpc
+      QPC_DBMS_PORT: 54321
+      QPC_DBMS_USER: qpc
+      QPC_SERVER_PASSWORD: q1w2e3r4
+      QPC_SERVER_TIMEOUT: 5
+      QPC_SERVER_USERNAME: admin
+      QPC_SERVER_PORT: "9443"
+      USE_SUPERVISORD: "false"
+
+    volumes:
+      - .:/app
+      - ./deploy:/deploy
+    depends_on:
+      - qpc-db
+    network_mode: "host"
```

Comment:
It looks like "host" mode is not supported on Mac according to https://docs.docker.com/network/host/. Specifically, this statement -
>The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, or Docker EE for Windows Server.

However, this is not an issue for me, since Mac is not a supported platform.

Reply:
Mmm we could go back to the slirp4netns, then. Maybe [installing it](https://formulae.brew.sh/formula/slirp4netns) on mac would be enough. If not, its better than host network for our purposes.

Reply:
@gtanzillo would you prefer I rebase with autosquash before or after your review?